app/assets/v2/js/pages/new_bounty.js

```diff
@@ -264,11 +267,11 @@ $(document).ready(function(){
             var _paysTokens = !isETH;
             var bountyIndex = bounty.issueAndActivateBounty(
                 account,            // _issuer
-                expire_date,        // _deadline
+                mock_expire_date,   // _deadline
```

Comment:
In this case, we always want to use the `mock_expire_date` here?

Reply:
i believe so... unless we want to provide the user the optional of actually expiring their bounty.. which i think might be too many explanatory overhead.. maybe pixelant will disagree